pkg/gatherers/clusterconfig/gather_silenced_alerts_test.go

```diff
@@ -0,0 +1,70 @@
+package clusterconfig
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/openshift/insights-operator/pkg/record"
+	"k8s.io/client-go/rest"
+
+	"github.com/openshift/insights-operator/pkg/utils/marshal"
+)
+
+type mockAlertsClient struct {
+	data []byte
+}
+
+func (c *mockAlertsClient) RestClient(t *testing.T) *rest.RESTClient {
+	ts := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if c.data == nil {
+			w.WriteHeader(http.StatusNotFound)
+		} else {
+			w.WriteHeader(http.StatusOK)
+			// nolint: errcheck
+			w.Write(c.data)
```

Comment:
What is the reason to not check the error?

Reply:
Well, it should not return an error, 'cause the data will never fail to be written. But, I can add the error handling, useless, but not a big deal.